fairworkflows/fairworkflow.py

```diff
@@ -25,13 +25,14 @@ class FairWorkflow(RdfWrapper):
         Fair Workflows may be fetched from Nanopublications, or created through the addition of FairStep's.
     """
 
-    def __init__(self, description=None, uri=None):
+    def __init__(self, description=None, label=None, uri=None):
```

Comment:
It's actually kind of strange that we can have a None default label and descriptions right? I just did a quick test and this results in: rdflib.term.Literal('None'). And this will pas through our validation for description and label.

Maybe we should remove the defaults?

Reply:
Good catch... Yes, this should definitely not pass validation...